nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/fetcher/Fetcher.java

```diff
@@ -29,6 +29,8 @@
  */
 public abstract class Fetcher {
 
+    protected static final String HIGHLIGHT_CTX_DATA = "highlight";
```

Comment:
Make it public with a Javadoc, and maybe find a better class to hold it? Because it's used by callers interested in highlights when doing `getContextData`.